Makefile

```diff
@@ -1,10 +1,16 @@
-PROG=bin/rbac-manager bin/workloads-manager bin/vault-manager bin/theatre-envconsul bin/theatre-consoles
```

Comment:
There's loads of changes in this file that are totally orthogonal to kube-builder v2. Can we strip this and everything else like it out of this PR? Should make it much more manageable